vllm/config.py

```diff
@@ -120,15 +120,6 @@ def _verify_load_format(self) -> None:
             if load_format == "auto":
                 load_format = "pt"
 
-        # FIXME(woosuk): This is a temporary hack. Support safetensor weights.
```

Comment:
It seems it's not supported. Can we raise error and ask users to use safetensors?

Reply:
It is supported _in principle_, but the pt weights uploaded to huggingface hub do not follow the same format as the safetensor weights. If someone was to convert safetensor weights back to pt (or do the equivalent operation in another way), pt would work.

Where would be the best place to add a warning?

Reply:
Got ya. I think you can add it here (just like the deleted code). Also, I think it makes sense to raise an error until the pt weights are fixed by HF because otherwise users can be confused.

Reply:
sounds good, will update tomorrow

Reply:
@Yard1 To make a release tonight, I've added the check to this PR. Sorry for any disruption this may cause.